base/setup/reactos/lang/tr-TR.rc

```diff
@@ -203,88 +204,88 @@ END
 
-    IDS_COPYING  "Copying %s"
-    IDS_MOVING   "Moving %s to %s"
-    IDS_RENAMING "Renaming %s to %s"
-    IDS_DELETING "Deleting %s"
+    IDS_COPYING  "Kopyalanıyor %s"
+    IDS_MOVING   "Taşınıyor %s to %s"
+    IDS_RENAMING "Yeniden Adlandırılıyor %s to %s"
+    IDS_DELETING "Siliniyor %s"
 
-    IDS_PREPARE_PARTITIONS "Preparing partitions..."
-    IDS_PREPARE_FILES "Preparing the list of files to be copied, please wait..."
-    IDS_COPYING_FILES "Copying the files..."
+    IDS_PREPARE_PARTITIONS "Bölümler hazırlanıyor..."
+    IDS_PREPARE_FILES "Kopyalanacak dosyaların listesi hazırlanıyor, lütfen bekleyin..."
+    IDS_COPYING_FILES "Dosyalar kopyalanıyor..."
 
-    IDS_CREATE_REGISTRY "Creating the registry..."
-    IDS_UPDATE_REGISTRY "Updating the registry..."
+    IDS_CREATE_REGISTRY "Kayıt defteri oluşturuluyor..."
+    IDS_UPDATE_REGISTRY "Kayıt defteri güncelleniyor..."
 END
 
 // Note to translators: please refer to the corresponding usetup/lang/*.h translations.
 STRINGTABLE
 BEGIN
-    IDS_REG_DONE "Done"
-    IDS_REG_REGHIVEUPDATE "Updating registry hives"
-    IDS_REG_IMPORTFILE "Importing %s"
-    IDS_REG_DISPLAYSETTINGSUPDATE "Updating display registry settings"
-    IDS_REG_LOCALESETTINGSUPDATE "Updating locale settings"
-    IDS_REG_ADDKBLAYOUTS "Adding keyboard layouts"
-    IDS_REG_KEYBOARDSETTINGSUPDATE "Updating keyboard layout settings"
-    IDS_REG_CODEPAGEINFOUPDATE "Adding codepage information"
-    IDS_REG_UNKNOWN "Unknown status %d"
+    IDS_REG_DONE "Tamamlandı"
+    IDS_REG_REGHIVEUPDATE "Kayıt defteri güncelleniyor"
+    IDS_REG_IMPORTFILE "İçe Alınıyor %s"
+    IDS_REG_DISPLAYSETTINGSUPDATE "Ekran kayıt defteri ayarlarını güncelleme"
+    IDS_REG_LOCALESETTINGSUPDATE "Yerel ayarların güncellenmesi"
+    IDS_REG_ADDKBLAYOUTS "Klavye düzenleri ekleme"
+    IDS_REG_KEYBOARDSETTINGSUPDATE "Klavye düzeni ayarlarını güncelleme"
+    IDS_REG_CODEPAGEINFOUPDATE "Kod sayfası bilgisi ekleme"
+    IDS_REG_UNKNOWN "Bilinmeyen durum %d"
 END
 
 /* Error Strings */
 
 STRINGTABLE
 BEGIN
-    IDS_ERROR_INVALID_INSTALLDIR_CHAR_TITLE "Invalid character"
-    IDS_ERROR_INVALID_INSTALLDIR_CHAR "The only valid characters are:\n\
-alphanumericals (a-z, A-Z, 0-9), and\n . \\ - _\n\
-Spaces are not allowed."
+    IDS_ERROR_INVALID_INSTALLDIR_CHAR_TITLE "Geçersiz karakter"
+    IDS_ERROR_INVALID_INSTALLDIR_CHAR "Geçerli karakterler şunlardır:\n\
+alfanümerik (a-z, A-Z, 0-9), ve\n . \\ - _\n\
+Boşluklara izin verilmez."
 
-    IDS_ERROR_DIRECTORY_NAME_TITLE "Invalid installation path"
-    IDS_ERROR_DIRECTORY_NAME "The ReactOS installation path must follow the DOS 8.3 naming scheme, \
-and only contain letters, digits, dashes and periods. Spaces are not allowed."
+    IDS_ERROR_DIRECTORY_NAME_TITLE "Geçersiz kurulum yolu"
+    IDS_ERROR_DIRECTORY_NAME "ReactOS kurulum yolu DOS 8.3 adlandırma şemasını takip etmelidir, 
+\n ve yalnızca harf, rakam, tire ve nokta içerir. Boşluklara izin verilmez."
```

Comment:
This will fix the build because the old version lacked a \ at the end of the first line, and also had a superfluous space there:
```suggestion
    IDS_ERROR_DIRECTORY_NAME "ReactOS kurulum yolu DOS 8.3 adlandırma şemasını takip etmelidir, \
ve yalnızca harf, rakam, tire ve nokta içerir. Boşluklara izin verilmez."
```